src/ArrayOfArraysView.hpp

```diff
@@ -148,7 +148,7 @@ class ArrayOfArraysView
    *        isn't invoked, this usually occurs with template argument deduction.
    */
   LVARRAY_HOST_DEVICE CONSTEXPRFUNC inline
-  ArrayOfArraysView< T, INDEX_TYPE const, true > const & toViewC() const LVARRAY_RESTRICT_THIS
+  ArrayOfArraysView< T, INDEX_TYPE const, true > const & toViewSemiConst() const LVARRAY_RESTRICT_THIS
```

Comment:
`toViewSemiConst` is a little strange sounding. Would it make sense to give this a more descriptive name? For instance `toViewConstSizes()`?

Reply:
Yeah it's not a great name. It might be useful in the future to have a name that's consistent across the different classes, but `toViewConstSparsity` is a good for the `CRSMatrix` class.